receiver/mongodbatlasreceiver/events.go

```diff
@@ -219,6 +261,44 @@ func (er *eventsReceiver) transformEvents(now pcommon.Timestamp, events []*mongo
 	return logs
 }
 
+func (er *eventsReceiver) transformOrgEvents(now pcommon.Timestamp, events []*mongodbatlas.Event, o *mongodbatlas.Organization) plog.Logs {
```

Comment:
This function is almost exactly the same as transformProjectEvents. Why not combine them?

Reply:
updated